package.json

```diff
@@ -18,18 +18,18 @@
     "@blockone/tslint-config-blockone": "^4.0.0",
     "@types/bunyan": "^1.8.6",
     "@types/dockerode": "^2.5.18",
-    "@types/jest": "23.3.13",
+    "@types/jest": "^24.0.25",
     "@types/massive": "^5.4.1",
     "@types/mongodb": "^3.1.4",
     "@types/node": "^10.5.1",
     "@types/node-fetch": "^2.1.6",
     "@types/request-promise-native": "^1.0.15",
-    "jest": "23.6.0",
-    "ts-jest": "23.10.5",
+    "jest": "^24.9.0",
+    "ts-jest": "^24.2.0",
     "tslint": "^5.10.0",
     "tslint-eslint-rules": "^5.3.1",
-    "typedoc": "^0.11.1",
-    "typescript": "^2.9.2"
+    "typedoc": "^0.15.6",
+    "typescript": "~3.5.3"
   },
   "dependencies": {
     "@types/express": "^4.16.1",
```

Comment:
While we're working on this code, let's version lock these "dependencies".  All "dependencies" should always be version locked.  "devDependencies" we don't impose the same restriction.